cmd/productinfo/main.go

```diff
@@ -66,6 +69,8 @@ func defineFlags() {
 	flag.StringSlice(providerFlag, []string{Ec2, Gce, Azure}, "Providers that will be used with the productinfo application.")
 	flag.String(azureSubscriptionId, "", "Azure subscription ID to use with the APIs")
 	flag.Bool(helpFlag, false, "print usage")
+	flag.Bool(metricsEnabledFlag, false, "if true get prometheus metrics, false by default")
+	flag.String(metricsPortFlag, ":9900", "prometheus metrics port")
```

Comment:
please change it to metricsAddressFlag, because it's not a simple port, but a bind address, and change the description to `the address where internal metrics are exposed`

Reply:
thx, fixed